.github/workflows/continuous_production.yml

```diff
@@ -21,13 +21,13 @@ jobs:
               go mod tidy
           fi
       - name: Build
-        run: go build -v .
+        run: make build
       - name: Run tests
         run: go test ./...
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          DYNAMODB_TABLE: dont-slack-evil-test-messages
+          DYNAMODB_TABLE_PREFIX: dont-slack-evil-test-
```

Comment:
You also need to update  the step "Create secrets file"

Reply:
Non justementNormalement non justement puisque cette directive est déjà dans le serverless.yml ?

Reply:
I see, yeah that makes sense.